### ISSUE TITLE
<b:commandButton> action doesn't fire managed bean function

### ISSUE THREAD

**Issue by rpinquie** [TheCoder4eu/BootsFaces-OSP#206]
I've got a command button defined as follows:
```xml
b:commandButton size="xs" ajax="true" look="primary" value="Update settings" update="panel" action="#{managerManagedBean.updateSettings}" icon="ui-icon-check"
```

When clicked, the button does not fire the server side "updateSettings" function of my managed bean "ManagerManagedBean". I am under 0.7.0 version - any idea why it doesn't work as it does with primefaces?

Cheers.

**Comment by stephanrauh** [comment]
At first glance, your code looks good. However, we've taken a new approach to AJAX with BootsFaces 0.8.0, which will be released soon. See http://bootsfaces.net/Staging/forms/ajax.jsf.

**Comment by rpinquie** [comment]
Yeah, it seems the problem arises when AJAX attributes "ajax" and "update" mainly are used. When do not use both, the sever side function is fired normally.

**Comment by stephanrauh** [comment]
Omitting `ajax="true"` or `update="something"` converts the AJAX request into an ordinary request. Truth to tell, until 0.7 BootsFaces offered only limited AJAX support. So please apology my reluctance to support this version. Please fetch the preview version of 0.8.0 (see #151 on how to get it), and read the new guide on AJAX (see http://bootsfaces.net/Staging/forms/ajax.jsf). We've already started to release this version, so using the developer preview version is almost no risk (unless you want to release your product tomorrow). Currently, we're doing the final tests.

**Comment by stephanrauh** [comment]
I didn't hear from you, so I consider this issue fixed (in the upcoming version 0.8.0 of BootsFaces). Please  reopen the issue if the problem persists with the new version.
